### PR TITLE
Instance/Stratholme - Move Maleki the Pallid to Eventmaps

### PR DESCRIPTION
**Pull request by Lumber-Jack-2**

### CODE REVIEW

File: src/server/scripts/EasternKingdoms/Stratholme/boss_maleki_the_pallid.cpp [hunk 1]
```diff
@@ -87,32 +87,37 @@ class boss_maleki_the_pallid : public CreatureScript
...
+            while (uint32 eventId = _events.ExecuteEvent())
             {
-                if (rand32() % 100 < 55)
-                    DoCastVictim(SPELL_DRAINLIFE);
-                DrainLife_Timer = 31000;
-            } else DrainLife_Timer -= diff;
+                switch (eventId)
+                {
+                    case EVENT_FROSTBOLT:
+                        if (rand() % 100 < 90)
```
Comment (DDuarte) [inline src/server/scripts/EasternKingdoms/Stratholme/boss_maleki_the_pallid.cpp]: change these again to rand32

Comment (Rushor) [inline src/server/scripts/EasternKingdoms/Stratholme/boss_maleki_the_pallid.cpp]: okay any idea how to change stuff without creating a new pr?

Comment (DDuarte) [inline src/server/scripts/EasternKingdoms/Stratholme/boss_maleki_the_pallid.cpp]: simply create a commit in your 3.3.5 branch

pull requests are nothing but branches, whatever changes you do to the branch you want to PR (in this case, `3.3.5`) will be reflected in this PR

File: src/server/scripts/EasternKingdoms/Stratholme/boss_maleki_the_pallid.cpp [hunk 2]
```diff
@@ -54,26 +61,19 @@ class boss_maleki_the_pallid : public CreatureScript
...
-            DrainLife_Timer = 31000;
-        }
-
-        InstanceScript* instance;
-
-        uint32 Frostbolt_Timer;
-        uint32 IceTomb_Timer;
-        uint32 DrainLife_Timer;
-
+        void Initialize() { }
```
Comment (Nyeriah) [inline src/server/scripts/EasternKingdoms/Stratholme/boss_maleki_the_pallid.cpp]: You don't need this

**Comment by Nyeriah** [comment]
Also, purely cosmetic but since you're gonna have to rebase to squash these commits, we follow the component/subcomponent standard when naming commits. For example, "Scripts/Stratholme: Move Maleki the Pallid to Eventmaps" so in this case the component being used is Scripts

**Pull request closed without merging**